src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -7,39 +7,34 @@ cssPrefix: pf-v5-l-flex
 
 ## Introduction
-The flex layout is based on the CSS Flex properties where the layout determines how a flex item will grow or shrink to fit the space available in its container. The system relies on a default spacer value `--pf-v5-l-flex--spacer--base`, whose value is `--pf-v5-global--spacer--md` or `16px` that is applied to flex items. By default, `flex-wrap` is set to `wrap` and `align-items` is set to `baseline`.
+The flex layout is based on the CSS Flex properties where the layout determines how a flex item will grow or shrink to fit the space available in its container. By default, a flex layout uses the [spacing system](#spacing-system) to space children along the main axis and [gap spacing](#gap-spacing) to space any wrapping children along the cross axis, `flex-wrap` is set to `wrap`, and `align-items` is set to `baseline`. The flex layout is infinitely nestable and can be nested to group items within.
 
-### Default spacing
-- Flex items (not last child): `margin-right: 16px`.
-- Nested `.pf-v5-l-flex` containers (not last child): `margin-right: 16px`.
-- `.pf-m-column` direct descendants (not last child): `margin-bottom: 16px`.
-- `.pf-m-column` nested `.pf-v5-l-flex` containers (not last child): `margin-bottom: 16px`.
-
-## Features
-
-- `.pf-v5-l-flex` is infinitely nestable and can be used to group items within.
-- `.pf-m-spacer-{xs,sm,md,lg,xl,2xl,3xl}` can be applied to parent or direct children and changes the spacer value for only the element to which it is applied. Responsive spacers can be used by appending `{-on-[breakpoint]}` to `.pf-m-spacer-{size}`. Example: `.pf-m-spacer-lg-on-xl`.
-- `.pf-m-space-items-{xs,sm,md,lg,xl,2xl,3xl}` can be applied to `.pf-v5-l-flex` only and changes the spacing of direct children only. Responsive spacers can be used by appending `{-on-[breakpoint]}` to `.pf-m-space-items-{size}`. Example: `.pf-m-space-items-lg-on-xl`.
+### Spacing
+The flex layout provides to ways of spacing its direct children.
+- [Spacing system](#spacing-system)
+  - The spacing system applies a margin between the flex layout's direct children to create space between items along the main axis only. The benefit of the spacing system is that it allows for variable spacing between direct children. Use the spacing system when individual flex items along the main axis require a different spacer than sibling items.
+- [Gap spacing](#gap-spacing)
+  - Gap spacing uses flex `gap` to space the flex layout's direct children, and can be applied to space rows (`row-gap`), columns (`column-gap`), or both (`gap`). The benefit of gap spacing is that item wrapping is improved and improved item spacing that works better with CSS flex's logical layout properties. E.g., spacing in RTL layouts that rely on logical properties is improved. Use the gap system when all direct children should use the same spacer for rows, columns, or both. **Note** using `gap` along the main axis will override any other spacing applied using the spacing system.
```

Comment:
Nit: feel like the "Note" gets kind of lost in the shuffle since this bullet is so lengthy:

```suggestion
  - Gap spacing uses flex `gap` to space the flex layout's direct children, and can be applied to space rows (`row-gap`), columns (`column-gap`), or both (`gap`). The benefit of gap spacing is that item wrapping is improved and improved item spacing that works better with CSS flex's logical layout properties. E.g., spacing in RTL layouts that rely on logical properties is improved. Use the gap system when all direct children should use the same spacer for rows, columns, or both. 
  - **Note** using `gap` along the main axis will override any other spacing applied using the spacing system.
```
Though both this and the Spacing sub-bullet could be broken up into several sub-bullets as well.